source/includes/_commands.md

```diff
@@ -1,22 +1,20 @@
 # Commands
 
-<aside class="warning">
-Currently, docs are updated. Sections with contraints and validation parts are gradually updated.
-</aside>
+A command changes the state, called World State View, by performing an action over an entity (asset, account) in the system. Any command should be included in a transaction to perform an action.
 
 ## Add asset quantity
 
 ### Purpose
 
-Purpose of _add asset quantity_ is to increase amount of assets on some account.
+The purpose of _add asset quantity_ is to increase the quantity of an asset on account of transaction creator. Use case scenario is to increase the number of mutable asset in the system, which can act as a claim on a commodity (e.g. money, gold, etc.)
```

Comment:
Maybe `_add asset quantity_ command` will be more readable

Reply:
Did you mean `number of mutable assets`?

Reply:
I meant a singular form, like USD or JPY asset let's say